.changeset/gentle-tomatoes-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sap/guided-answers-extension-webapp": patch
```

Comment:
I'd say this is a `minor` change as the behavior of the app changes. However, we could also argue that the feature is behind a toggle for now.